### ISSUE TITLE
TXT template variable not substituted if within angle brackets

### ISSUE THREAD

**Issue by ethauvin** [rife2/rife2-core#5]
For example:

```
public final class {{v className/}} implements Comparable<{{v className/}}> {
```

is substituted as:

```
public final class MyClass implements Comparable<{{v className/}}> {
```

Adding a space after and before the angle brackets works around the issue:

```
public final class {{v className/}} implements Comparable< {{v className/}} > {
```
is substituted as:

```
public final class MyClass implements Comparable< MyClass > {
```

**Comment by gbevin** [comment]
Is this with the latest RIFE2 1.8.0 snapshot? I fixed something similar two weeks ago https://github.com/rife2/rife2-core/commit/301339449ec290cf118d969ef2b51ee02b8b83b7

**Comment by ethauvin** [comment]
You're right, it is fixed in bld 2.0.0-SNAPSHOT.
